.travis.yml

```diff
@@ -2,8 +2,14 @@ sudo: false
 language: node_js
 cache: yarn
 script: npm run test:ci
+dist: trusty
```

Comment:
When using the newer trusty images on TravisCI (https://docs.travis-ci.com/user/reference/trusty/) we can run on the latest Firefox on PRs at least.